src/test/ui/closures/2229_closure_analysis/arrays-completely-captured.rs

```diff
@@ -0,0 +1,19 @@
+#![feature(capture_disjoint_fields)]
+//~^ WARNING the feature `capture_disjoint_fields` is incomplete
+#![feature(rustc_attrs)]
+
+// Ensure that capture analysis results in arrays being completely captured.
+fn main() {
+    let mut m = [1, 2, 3, 4, 5];
+
+    let mut c = #[rustc_capture_analysis]
+    //~^ ERROR: attributes on expressions are experimental
+    || {
+        m[0] += 10;
+        //~^ ERROR: Capturing m[] -> MutBorrow
+        //~^^ ERROR: Min Capture m[] -> MutBorrow
```

Comment:
```suggestion
        //~| ERROR: Min Capture m[] -> MutBorrow
```
you can use this notation too